CombinatorialGames/Game/Special.lean

```diff
@@ -19,7 +19,7 @@ universe u
 
 namespace PGame
 
-/-! ### Star -/
+/-! ## Star -/
```

Comment:
As I understand it, the convention on heading levels within Mathlib is:

1. Only for the title in the module docstring.
2. For sections within the module docstring.
3. For subsections of the file

So I believe `###` was correct.

Reply:
Oh! Interesting :+1: